### ISSUE TITLE
Can't get the .exe file to do anything.

### ISSUE THREAD

**Issue by Zlirra** [gameboy9/7thSagaRando#3]
After downloading the .exe file and the source code, I was unable to get the .exe file to execute anything. Double click on it and nothing happens. Run as admin, nothing happens. Turned down security settings, nothing happened. Try putting it in with other parts of the source code folders, nothing happens.

Now is this just me overlooking something super simple or is there actually something wrong with the .exe file?

**Comment by gameboy9** [comment]
Just to confirm... did you download the EXE from this location?  https://github.com/gameboy9/7thSagaRando/releases/tag/1.3

You may also need to install .NET Framework 4.0 or 4.5.

**Comment by Zlirra** [comment]
My apologies. I met to get back to that message.

To answer your question, yes I downloaded it from your github.

The solution is, I think because I was running it on Windows 10, the
windows security didn't allow the .exe to run properly. To solve that I
right clicked on the .exe file and went to properties. At the very bottom
there is a check box that needed to be checked to 'unblock'. The message
stated that it was being blocked because it was sent from another computer.
Basically Windows security was blocking it.

Thanks for getting back to me though. An absolute threat. Thanks for your
hard work

**Comment by Kronoan** [comment]
I unchecked the box too just to make sure and it worked! Thanks for the solution Zlirra!
